src/App.js

```diff
@@ -15,6 +15,41 @@ import * as actions from './store/actions';
 import  { Redirect } from 'react-router-dom';
 
 
+import firebase from 'firebase';
+var config = {
+    apiKey: "AIzaSyBibrWw6b833N4ar2-wY5wIT3TT1IR_pyg",
+    authDomain: "saharok-221817.firebaseapp.com",
+    databaseURL: "https://saharok-221817.firebaseio.com",
+    projectId: "saharok-221817",
+    storageBucket: "saharok-221817.appspot.com",
+    messagingSenderId: "406120965570"
+};
+firebase.initializeApp(config);
+
+const initializePush = () => {
+  const messaging = firebase.messaging();
+  messaging
+    .requestPermission()
+    .then(() => {
+    console.log('Permission is granted by user');
+    return messaging.getToken();
+    })
+    .then(token => {
+    console.log('FCM Token:', token);
+    // Write your logic to send token to your server
+    })
+    .catch(error => {
+    console.log('Error Occurred', error);
+    });
+}
```

Comment:
Было бы здорово, если найдется возможность поработать с отступами :)

```suggestion
const initializePush = () => {
  const messaging = firebase.messaging();
  messaging
    .requestPermission()
    .then(() => {
      console.log('Permission is granted by user');
      return messaging.getToken();
    })
    .then(token => {
      console.log('FCM Token:', token);
      // Write your logic to send token to your server
    })
    .catch(error => {
      console.log('Error Occurred', error);
    });
}
```